### PR TITLE
chore: bump fundamentals version and reference ie 11 compat css

### PR DESCRIPTION
**Pull request by mobot11**
### Description
Bump fundamentals dependency to version with IE 11 css and reference IE 11 css distribution.

fixes https://github.com/SAP/fundamental-react/issues/436

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ For an existing react application, follow the steps below:
 1. Load the fiori-fundamentals styles. If using create-react-app, this will be in `App.css`.
 
     ```scss
-    @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
+    @import '~node_modules/fiori-fundamentals/dist/fiori-fundamentals-ie11.min.css';
```
Comment (greg-a-smith) [inline README.md]: I thought the tilde (`~`) character already resolved to the project's `node_modules` directory.

Comment (mobot11) [reply README.md]: https://stackoverflow.com/questions/39535760/what-does-a-tilde-in-a-css-url-do Looks like you are right. Updating.

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ For an existing react application, follow the steps below:
 1. Load the fiori-fundamentals styles. If using create-react-app, this will be in `App.css`.
 
     ```scss
-    @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
+    @import '~fiori-fundamentals/dist/fiori-fundamentals-ie11.min.css';
```
Comment (greg-a-smith) [inline README.md]: This is probably a Fiori Fundamentals question, but why does the 1.4.3 distribution package have both the `-ie11` CSS file and the non-`-ie11` CSS file?  If it needs to support IE 11, why isn't _that_ just the only CSS file distributed?

Comment (mobot11) [reply README.md]: I wondered the same thing, see https://github.com/SAP/fundamental/issues/1244#issuecomment-459779074

**Comment by bcullman** [comment]
this is not my preferred way to support IE, but I am having trouble locating automated tooling to generate an IE-only CSS override file.

**Pull request merged**